runtime-config.md

```diff
@@ -34,6 +34,12 @@ Additional filesystems can be declared as "mounts", specified in the *mounts* ar
         "source": "/volumes/testing",
         "destination": "/data",
         "options": ["rbind","rw"]
+    },
+    {
+        "type": "ext3",
+        "source": "qcow2:/volumes/data.qcow2",
+        "destination": "/data",
```

Comment:
To keep a consistent example, it probably makes sense to either pick a different destination here (so we don't clobber the bind-mount) or break this hypervisor-specific example off into it's own section.  For example:

    …
         {
             "type": "bind",
             "source": "/volumes/testing",
             "destination": "/data",
             "options": ["rbind","rw"]
        }
    ]

*Example (Hypervisor)*

This is like the Linux example, except the final entry should look something like:

     {
         "type": "ext3",
         "source": "qcow2:/volumes/data.qcow2",
         "destination": "/data",
         "options": ["rw"]
    }